packages/web-components/src/components/va-text-input/va-text-input.tsx

```diff
@@ -276,9 +278,9 @@ export class VaTextInput {
                   {i18next.t('required')}
                 </span>
               )}
+              {hint && <span class="usa-hint">{hint}</span>}
```

Comment:
I have moved the hint text element inside of the `label` so that it is automatically read instead of using an `aria-describedby`. I thought this would give us one less id to have to keep track of (because there are a number of conditional aria-describedby id's).

The only downside is that the hint text is in a label condition. So if there is not a label, the hint will not display. I could not think of a scenario where hint text would be used without a label though so this seemed ok.

Reply:
Seems like a reasonable compromise to me.